docs/src/prototypes/VirtualizedTree/VirtualizedTree.tsx

```diff
@@ -32,6 +45,13 @@ function TreeVirtualizer(props: TreeVirtualizerProps) {
       height={300}
       rowCount={props.renderedItems.length}
       width={600}
+      scrollToIndex={scrollToIndex}
+      onRowsRendered={() => {
+        if (scrollToIndex !== undefined) {
+          props.renderedItems[scrollToIndex].props.contentRef.current.focus()
+          setScrollToIndex(undefined)
```

Comment:
Unfortunate implementation but should not cause a DOM render. Any better ideas?

Need to un-set it after this operation because if left it in the state then at each onRowsRendered that item will always be scrolled to.